cmd/nerdctl/apparmor_inspect_linux.go

```diff
@@ -37,10 +38,7 @@ func newApparmorInspectCommand() *cobra.Command {
 }
 
 func apparmorInspectAction(cmd *cobra.Command, args []string) error {
-	b, err := apparmor.DumpDefaultProfile(defaults.AppArmorProfileName)
-	if err != nil {
-		return err
-	}
-	_, err = fmt.Fprint(cmd.OutOrStdout(), b)
-	return err
+	options := &types.InspectCommandOptions{}
+	options.Writer = cmd.OutOrStdout()
+	return apparmor.InspectAction(options)
```

Comment:
I don't think the `Action` suffix is still needed here. `Action` is mainly for `cobra`. Maybe just `apparmor.Inspect`

Reply:
SGTM